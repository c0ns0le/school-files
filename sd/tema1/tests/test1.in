5
OPEN_WINDOW 4 36218 47912 88 3435196
OPEN_WINDOW 2 4868 15854 671 7438887
OPEN_WINDOW 1 15141 29381 1034 25204572
ADD_PERSON 47865 23281 1
ADD_PERSON 15490 5613 2
ADD_PERSON 4492 16317 1
PRINT_STACKS
ADD_PERSON 56696 46582 2
OPEN_WINDOW 3 2574 15793 826 7865808
FLUSH_STACKS
PRINT_STACKS
PRINT_STACKS
PROCESS 1 2
PRINT_QUEUES
FLUSH_STACKS
ADD_PERSON 65197 27438 1
ADD_PERSON 10853 22764 1
ADD_PERSON 10970 4999 3
PROCESS_ALL 1
PROCESS_ALL 4
ADD_PERSON 91404 44431 4
PROCESS 4 1
ADD_PERSON 37294 7817 2
PROCESS_ALL 1
FLUSH_STACKS
ADD_PERSON 2660 25640 1
FLUSH_STACKS
FLUSH_STACKS
PRINT_QUEUES
PRINT_QUEUES
ADD_PERSON 90329 13895 3
PROCESS 3 1
ADD_PERSON 70717 21133 1
OPEN_WINDOW 0 43936 58653 336 17200314
PRINT_QUEUES
