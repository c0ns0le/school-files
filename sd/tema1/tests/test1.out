STACKS:
1: 
2: 
4: 
FLUSHING:
1: 
2: 
3: 
4: 
STACKS:
1: 
2: 
3: 
4: 
STACKS:
1: 
2: 
3: 
4: 
QUEUES:
1: 
2: (15490, 5613), (56696, 46582)
3: 
4: 
FLUSHING:
1: 16317, 23281
2: 
3: 
4: 
FLUSHING:
1: 22764, 27438
2: 
3: 
4: 44431
FLUSHING:
1: 
2: 
3: 
4: 
FLUSHING:
1: 
2: 
3: 
4: 
QUEUES:
1: (2660, 25640)
2: (15490, 5613), (56696, 46582), (37294, 7817)
3: (10970, 4999)
4: 
QUEUES:
1: (2660, 25640)
2: (15490, 5613), (56696, 46582), (37294, 7817)
3: (10970, 4999)
4: 
QUEUES:
0: 
1: (2660, 25640), (70717, 21133)
2: (15490, 5613), (56696, 46582), (37294, 7817)
3: (90329, 13895)
4: 
